api/src/main/java/org/openmrs/module/cohort/CohortMember.java

```diff
@@ -1,5 +1,8 @@
 package org.openmrs.module.cohort;
 
+import org.openmrs.BaseOpenmrsData;
```

Comment:
You might want to check your IDEs import order settings. See [here](https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+IntelliJ#DeveloperHow-ToSetupAndUseIntelliJ-JavaImports).

Reply:
ah, I don't know why I still have two versions of the IDE. I should just delete `ce`. Thanks, good spot.